string name
mission_control/TaskData data
---
bool success
string message